.github/CODEOWNERS

```diff
@@ -386,25 +386,25 @@ lerna.json @grafana/frontend-ops
 /public/app/plugins/panel/alertGroups/ @grafana/alerting-squad-frontend
 /public/app/plugins/panel/alertlist/ @grafana/alerting-squad-frontend
 /public/app/plugins/panel/annolist/ @grafana/user-essentials
-/public/app/plugins/panel/barchart/ @grafana/grafana-bi-squad
+/public/app/plugins/panel/barchart/ @grafana/grafana-bi-squad @grafana/dataviz-squad
```

Comment:
If I am understanding correctly `barchart` will be shared between DataViz + BI?

Reply:
It looks like yes.